CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 Changes by Version
 ==================
 
+0.54.0
+-------------------
+### 💡 Enhancements 💡
+* Add a simple test to ensure we are opening the correct ports for otlp… ([#945](https://github.com/open-telemetry/opentelemetry-operator/pull/945), [@kevinearls](https://github.com/kevinearls))
```

Comment:
Let's remove this it is not a user-facing issue.

Reply:
Done